lib/views/loading_screen.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'home.dart';
+import '../models/repository/genre_repository.dart';
+import '../models/repository/movie_repository.dart';
+
+class LoadingScreen extends StatefulWidget {
+  const LoadingScreen({super.key});
+
+  @override
+  State<LoadingScreen> createState() => _LoadingScreenState();
+}
+
+class _LoadingScreenState extends State<LoadingScreen> {
+  Widget _body = const SafeArea(
+    child: Center(
+      child: SizedBox(
+        child: CircularProgressIndicator(),
+      ),
+    ),
+  );
+
+  void _loadNextScreen() async {
+    await MovieRepository().fetchData();
+    await GenreRepository().fetchData();
+    setState(
+      () {
+        _body = const Home();
+      },
+    );
```

Comment:
I'd use the `Navigator` instead of this

Reply:
I removed the view. I guess I can say it's done